src/code-gen/markup-example.js

```diff
@@ -22,7 +22,8 @@ let mkExampleMarkup = (markup, model) => `
         var player = document.querySelector('pie-player');
 
         player.addEventListener('pie-player-ready', function(event){
-          player.controller = new pie.Controller(model.pies, pie.controllerMap);
+          var scoringProcessor = new pie.ScoringProcessorFactory().getProcessor(model);           
```

Comment:
I didn't realize when this was mentioned earlier that this would be something the user had to config - i don't think this should be exposed to the user, I feel like it should be embedded within the controller.